### PR TITLE
Cleaup previous widevine vmp PRs

### PR DESCRIPTION
**Pull request by simonhong**
Remove/reduce unnecessary patches with our scripts.
Remove brave_enable_cdm_host_verification gn vars.

Issue: https://github.com/brave/brave-browser/issues/4905
Related PR: https://github.com/brave/brave-browser/pull/4909

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [x] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [x] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by simonhong** [comment]
Set skip labels except Windows because all others are passed and only Win64 was failed due to installer signing. it was also unrelated with this PR.

File: patches/chrome-tools-build-win-create_installer_archive.py.patch [hunk 1]
```diff
@@ -1,92 +1,33 @@
...
-+  """Copies already signed three binaries - brave.exe, chrome.dll and chrome_child.dll
-+  These files are signed during the build phase to create widevine sig files.
-+  """
-+  src_dir = os.path.join(output_dir, 'signed_binaries')
-+  chrome_dir = os.path.join(staging_dir, CHROME_DIR)
-+  version_dir = os.path.join(chrome_dir, current_version)
-+  shutil.copy(os.path.join(src_dir, 'brave.exe'), chrome_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome.dll'), version_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome_child.dll'), version_dir)
++  from create_installer_archive_helper import (
```
Comment (bridiver) [inline patches/chrome-tools-build-win-create_installer_archive.py.patch]: please put on one line, lint is not an issue for patches

File: patches/chrome-tools-build-win-create_installer_archive.py.patch [hunk 2]
```diff
@@ -1,92 +1,33 @@
...
    """Copies installer archive files specified in section from src_dir to
-@@ -531,6 +596,11 @@ def main(options):
+@@ -531,6 +536,12 @@ def main(options):
      version_numbers = prev_version.split('.')
      prev_build_number = version_numbers[2] + '.' + version_numbers[3]
  
 +  if not options.skip_signing:
 +    from sign_binaries import sign_binaries
 +    sign_binaries(staging_dir)
++    from create_installer_archive_helper import CopyPreSignedBinaries
```
Comment (bridiver) [inline patches/chrome-tools-build-win-create_installer_archive.py.patch]: this should go at https://github.com/brave/brave-core/pull/2693/files#diff-a9c9a8da7aa4df821394352a0ca04a27R9 with the others, but also we shouldn't be calling two methods here. We should have a single method for both `sign_binaries` and `CopyPreSignedBinaries`

File: patches/chrome-tools-build-win-create_installer_archive.py.patch [hunk 3]
```diff
@@ -1,92 +1,33 @@
...
-+  src_dir = os.path.join(output_dir, 'signed_binaries')
-+  chrome_dir = os.path.join(staging_dir, CHROME_DIR)
-+  version_dir = os.path.join(chrome_dir, current_version)
-+  shutil.copy(os.path.join(src_dir, 'brave.exe'), chrome_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome.dll'), version_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome_child.dll'), version_dir)
++  from create_installer_archive_helper import (
++      CopyBraveExtensionLocalization,
++      CopyBraveRewardsExtensionLocalization)
++  CopyBraveExtensionLocalization(config, staging_dir, g_archive_inputs)
```
Comment (bridiver) [inline patches/chrome-tools-build-win-create_installer_archive.py.patch]: these two methods should be combined. There is no reason to call two methods in a row in patch that have the same params

**Comment by simonhong** [comment]
@bridiver Reduced to just 3 lines ;) PTAL again.

File: patches/chrome-tools-build-win-create_installer_archive.py.patch (hunk 3 above)
Comment (simonhong) [reply patches/chrome-tools-build-win-create_installer_archive.py.patch]: done.

File: patches/chrome-tools-build-win-create_installer_archive.py.patch (hunk 2 above)
Comment (simonhong) [reply patches/chrome-tools-build-win-create_installer_archive.py.patch]: done.

File: patches/chrome-tools-build-win-create_installer_archive.py.patch (hunk 1 above)
Comment (simonhong) [reply patches/chrome-tools-build-win-create_installer_archive.py.patch]: done.

**Comment by simonhong** [comment]
Also brave extension & rewards extension localization works properly.

**Pull request merged**